nss-day-cohort-47/tabloid-cli-national-enquirer
# Dcu soft delete

# Description
Fixed blog to have soft delete.

## Type of change
Please delete options that are not relevant.

# Testing Instructions
Test blog soft delete by removing a blog and then checking the database to make sure isDeleted is true for that blog.
# Checklist:
